dev/src/Command/AddComponentCommand.php

```diff
@@ -184,21 +210,215 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $composer->updateMainComposer();
         $composer->createComponentComposer($new->displayName, $new->githubRepo);
 
+        $this->validateOptions($input, $output);
+        if ($input->getOption('bazel-path')) {
+            $googleApisDir = realpath($input->getOption('bazel-path'));
+            $output->writeln("\n\nbazel build library");
+            $output->writeln($this->bazelQueryAndBuildLibrary($new->protoPath, $googleApisDir));
+            $output->writeln("\n\nCopying the library code from bazel-bin");
+            $output->writeln($this->owlbotCopyBazelBin($new->componentName, $googleApisDir));
+            $output->writeln("\n\nOwlbot post processing");
+            $output->writeln($this->owlbotPostProcessor());
+        } else if ($input->getOption('googleapis-gen-path')) {
+            $this->checkDockerAvailable();
+            $googleApisGenDir = realpath($input->getOption('googleapis-gen-path'));
+            $output->writeln("\n\nCopying the library code from googleapis-gen");
+            $output->writeln($this->owlbotCopyCode($new->componentName, $googleApisGenDir));
+            $output->writeln("\n\nOwlbot post processing");
+            $output->writeln($this->owlbotPostProcessor());
+        }
+
         $output->writeln('');
         $output->writeln('');
         $output->writeln('Success!');
 
         return 0;
     }
 
+    private function validateOptions(InputInterface $input, OutputInterface $output): void
+    {
+        $error = '<error>googleapis-gen-path and bazel-path options ' .
```

Comment:
nit: it would be better to either put the `$error` variable inside the conditional, or just supply the string directly as the argument to `writeln`